### DISCUSSION TITLE
Pls add me to the project (rubiidev-ipad | alt of rubiidev)

### DISCUSSION THREAD

**Discussion by Rubiidev-IPad** [cytrinc/bark-coding#176] [Applications] [answered]
i'm a alt of @Rubiidev-18 and @rubiidev18alt here because i don't have credentials for those 2 accounts on the ipad im using this account on.

(also we def need another cat pfp)

**Comment by raynepaws** [comment]
just log in on one of your other accounts...

> i don't have credentials for those 2 accounts on the ipad

so get them

(sorry if i sound rude)

**Comment by mariocraft987** [comment, marked as answer]
ummm.. sorry, no
